### PR TITLE
ODATA-1631 - Validation.Constraint also on NavigationProperty

### PR DESCRIPTION
**Pull request by ralfhandl**

### CODE REVIEW

File: vocabularies/Org.OData.Validation.V1.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@
         <Annotation Term="Core.Description" String="The value of the annotated property, parameter, or term must be an integer multiple of this positive value. For temporal types, the value is measured in seconds." />
       </Term>
 
-      <Term Name="Constraint" Type="Validation.ConstraintType" Nullable="false" AppliesTo="Property EntityType ComplexType">
+      <Term Name="Constraint" Type="Validation.ConstraintType" Nullable="false" AppliesTo="Property NavigationProperty EntityType ComplexType">
```
Comment (HeikoTheissen) [inline vocabularies/Org.OData.Validation.V1.xml:112]: Can this also apply to a Term?

Comment (HeikoTheissen) [reply vocabularies/Org.OData.Validation.V1.xml:112]: Sorry, I mixed this up with other capabilities terms, like `Minimum`.

**Review by HeikoTheissen — APPROVED** [review]

File: vocabularies/Org.OData.Validation.V1.xml (hunk 1 above)
Comment (ralfhandl) [reply vocabularies/Org.OData.Validation.V1.xml:112]: I'm not opposed to adding `Parameter` and `Term` just to be consistent with the specialized constraints: should we do that now, or wait for a concrete use case?

Comment (HeikoTheissen) [reply vocabularies/Org.OData.Validation.V1.xml:112]: `Constraint` is about error messages presented to the user. This makes sense for `Parameter` (which the user may have entered on the UI), but not for `Term`.

Comment (ralfhandl) [reply vocabularies/Org.OData.Validation.V1.xml:112]: Added `Parameter`

**Review by mikepizzo — APPROVED** [review]

**Pull request merged**